### ISSUE TITLE
Аномально высокая влажность в туалете

### ISSUE THREAD

**Issue by Komzpa** [RudenkoPeter/KD#10]
Зимовка показывает, что в туалете влажность больше, чем на улице. Нужно осушать.

![image](https://cloud.githubusercontent.com/assets/810638/21560832/bf411d24-ce75-11e6-8711-6737d03b5d97.png)

**Comment by peter-hanzo** [comment]
На данный момент рассматриваю три причины: забор влажности из труб канализации или щелей пола, усугубляет отсутствие проветривания и вентиляции.
1. Налажу проветривание
2. Уберу источники влажности

**Comment by peter-hanzo** [comment]
В ближайшее время сделаю временные отверстия для проветривания, а после ремонта двери можно будет открыть окно.

**Comment by Komzpa** [comment]
есть хоть какая-нибудь подтверждающая математика? :)

**Comment by peter-hanzo** [comment]
Подтверждающая что? Что если открыть окно влажность упадет?

**Comment by Komzpa** [comment]
да. с пониманием, откуда она берётся, и при каких параметрах начнёт проявляться внутри дома.

**Comment by peter-hanzo** [comment]
Предварительно причины три:
1. Дождевая вода затекает под санузел
2. Испарения воды попадают в санузел через щели в полу
3. Влажность накапливается в помещении т.к. нет проветривания

Усугубляет ситуацию северное расположение санузла т.е. солнце нагревает только часть стены и дверь, а основной периметр находится постоянно в тени.

В доме такого не будет т.к.:

1. Пол герметичен и утеплен
2. Вода под дом затекает, но намного меньше чем под санузел
3. Высота помещения помогает циркуляции воздуха
4. Солнце нагревает помещение через большое окно

**Comment by peter-hanzo** [comment]
5 марта открыл верхнюю часть обшивки для проветривания + переставил датчик влажности в центр помещения. Раньше он лежал на полу.
